packages/x-components/src/components/page-selector.vue

```diff
@@ -0,0 +1,226 @@
+<template>
+  <nav class="x-flex x-items-center x-justify-center x-gap-2" aria-label="Pagination">
```

Comment:
We can't use tailwindcss classes here, specially with the x prefix. If we do we will force the consumer to have the same prefix.